src/ir/inst.rs

```diff
@@ -412,13 +478,15 @@ impl<'a, 'b> InstBuilder<'a, 'b> {
         self.build(data, void_ty())
     }
 
+    /// Creates sig instruction to instantiates a proces or entity
```

Comment:
I think this should read:
```suggestion
    /// Creates sig instruction to allocate a signal
```